dirtify.py

```diff
@@ -104,37 +87,90 @@
-        dirty = cv2.GaussianBlur(dirty, (args.blur, args.blur), 0)
+        img = cv2.GaussianBlur(in_img, (args.blur, args.blur), 0)
         
     if args.rblur is not None:
+        # Apply a radial blur
         radius = 0.01
-
-        growMapx = np.abs(np.tile(np.arange(out_h) + ((np.arange(out_h) - mid_w) * radius), (out_w, 1)).astype(np.float32))
-        shrinkMapx = np.abs(np.tile(np.arange(out_h) - ((np.arange(out_h) - mid_w) * radius), (out_w, 1)).astype(np.float32))
-        growMapy = np.abs(np.tile(np.arange(out_w) + ((np.arange(out_w) - mid_h) * radius), (out_h, 1)).transpose().astype(np.float32))
-        shrinkMapy = np.abs(np.tile(np.arange(out_w) - ((np.arange(out_w) - mid_h) * radius), (out_h, 1)).transpose().astype(np.float32))
+        mid_y = h / 2
+        mid_x = w / 2
+        growMapx = np.abs(np.tile(np.arange(h) + ((np.arange(h) - mid_x) * radius), (w, 1)).astype(np.float32))
+        shrinkMapx = np.abs(np.tile(np.arange(h) - ((np.arange(h) - mid_x) * radius), (w, 1)).astype(np.float32))
+        growMapy = np.abs(np.tile(np.arange(w) + ((np.arange(w) - mid_y) * radius), (h, 1)).transpose().astype(np.float32))
+        shrinkMapy = np.abs(np.tile(np.arange(w) - ((np.arange(w) - mid_y) * radius), (h, 1)).transpose().astype(np.float32))
         
         for i in range(args.rblur):
-            tmp1 = cv2.remap(dirty, growMapx, growMapy, cv2.INTER_LINEAR, borderMode=cv2.BORDER_REFLECT)
-            tmp2 = cv2.remap(dirty, shrinkMapx, shrinkMapy, cv2.INTER_LINEAR, borderMode=cv2.BORDER_REFLECT)
-            dirty = cv2.addWeighted(tmp1, 0.5, tmp2, 0.5, 0)
+            tmp1 = cv2.remap(in_img, growMapx, growMapy, cv2.INTER_LINEAR, borderMode=cv2.BORDER_REFLECT)
+            tmp2 = cv2.remap(in_img, shrinkMapx, shrinkMapy, cv2.INTER_LINEAR, borderMode=cv2.BORDER_REFLECT)
+            img = cv2.addWeighted(tmp1, 0.5, tmp2, 0.5, 0)
     
     if args.plus:
         # superimpose a white plus at a random position
-        x = round(random.random() * out_w)
-        y = round(random.random() * out_h)
-        cv2.line(dirty, (0,y), (out_w, y), (255,255,255), 1)
-        cv2.line(dirty, (x,0), (x, out_h), (255,255,255), 1)
+        x = round(random.random() * w)
+        y = round(random.random() * h)
+        img = in_img # make a copy so we don't overdraw the clean image
+        cv2.line(img, (0,y), (w, y), (255,255,255), 1)
+        cv2.line(img, (x,0), (x, h), (255,255,255), 1)
     
     if args.xout:
         # Draw a dark red X through the image
-        cv2.line(dirty, (0,0    ), (out_w,out_h), (0,0,64), 2)
-        cv2.line(dirty, (0,out_h), (out_w,0    ), (0,0,64), 2)
+        img = in_img # make a copy so we don't overdraw the clean image
+        cv2.line(img, (0,0), (w,h), (0,0,64), 2)
+        cv2.line(img, (0,h), (w,0), (0,0,64), 2)
     
-    cv2.imwrite(f"{dirty_dir}/{out_name}", dirty)
+    return img
+
+
+# Step through images
+for i, clean in tqdm(enumerate(images), total=len(images)):
```

Comment:
Can we just do `for clean in tqdm(images, total=len(images))` and then replace `images[i]` with `clean` in the rest of this block?

Reply:
Maybe? This is working, but making that change yields an error. Let's just leave it as is, and if we want to clean that up, it could be a separate commit.

Reply:
Sure -- small commits are preferable anyway.